tinylog-impl/src/main/java/org/tinylog/impl/format/pattern/placeholders/ContextPlaceholder.java

```diff
@@ -33,13 +42,36 @@ public ValueType getType() {
 
     @Override
     public String getValue(LogEntry entry) {
-        return entry.getContext().get(key);
+        return key == null ? contextKeysAndValuesOrDefault(entry, null) : entry.getContext().get(key);
     }
 
     @Override
     public void render(StringBuilder builder, LogEntry entry) {
-        String value = entry.getContext().getOrDefault(key, "");
-        builder.append(value);
+        if (key != null) {
+            String value = entry.getContext().getOrDefault(key, "");
+            builder.append(value);
+        } else {
+            builder.append(contextKeysAndValuesOrDefault(entry, ""));
+        }
+    }
+
+    /**
+     * Creates a string representation of all keys and values in thread context.
+     *
+     * @param entry the log entry whose thread context is to be represented as a string
+     * @param defaultValue the value to return if the thread context has no keys or values
+     * @return a string representation of all keys and values in entry's thread context
+     */
+    private static String contextKeysAndValuesOrDefault(LogEntry entry, String defaultValue) {
```

Comment:
Could you start Javadoc descriptions with an upper letter? This makes it easy to distinguish the actual param from its description.

```
* @param entry The log entry whose thread context is to be represented as a string
* @param defaultValue The value to return if the thread context has no keys or values
* @return A string representation of all keys and values in entry's thread context
```

Reply:
Done.